sql/core/src/test/scala/org/apache/spark/sql/execution/PlannerSuite.scala

```diff
@@ -994,6 +994,38 @@ class PlannerSuite extends SharedSparkSession with AdaptiveSparkPlanHelper {
       }
     }
   }
+
+  test("Remove redundant shuffle exchange") {
```

Comment:
Shall we add a JIRA prefix in the test title?

Reply:
I labeled this as an `improvement`. Do we need add the prefix, or do we need to label this as a `bug`?

Reply:
Added the prefix just in case.